### PR TITLE
Test that data type lexical form pass the data type check

### PR DESCRIPTION
**Pull request by MFSY**
"All value nodes of a property must be subclasses of a specific root class."

Just to demo the idea.

### CODE REVIEW

File: modules/shacl/src/main/scala/es/weso/shacl/SHACLPrefixes.scala [hunk 1]
```diff
@@ -32,6 +32,7 @@ object SHACLPrefixes {
   lazy val sh_oneOrMorePath: IRI = sh + "oneOrMorePath"
 
   lazy val sh_class : IRI= sh + "class"
+  lazy val sh_rootClass : IRI= sh + "rootClass"
```
Comment (labra) [inline modules/shacl/src/main/scala/es/weso/shacl/SHACLPrefixes.scala]: I think it would be better if we could use a different namespace than `sh`

Comment (MFSY) [inline modules/shacl/src/main/scala/es/weso/shacl/SHACLPrefixes.scala]: I agree as I said [here](https://github.com/labra/shaclex/issues/41). I see that the TopQuandrant SHACL implementation have [rootClass](http://shacl.org/playground/dash.ttl) constraint under dash (http://datashapes.org/dash#) namespace.
I have no idea about how this is handled usually. I would suggest to use shext (http://www.w3.org/ns/shacl/ext#).

Thoughts ?

**Comment by MFSY** [comment]
I made another one (Test that data type lexical form pass the data type check) testing data type checking. But since it's from the same branch (master of my fork) github considered them as being part of the same pull request.

I suggest you close this pull request, so that I can make it again but in a different branch.

**Comment by MFSY** [comment]
I made another one (Test that data type lexical form pass the data type check) testing data type checking. But since it's from the same branch (master of my fork) github considered them as being part of the same pull request.

I will make an other pull request (for rootClass) once the data type related one is closed.

**Pull request closed without merging**